srcpkgs/pass-import/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'pass-import'
 pkgname=pass-import
-version=3.2
-revision=2
+version=3.3
+revision=1
 build_style=python3-module
 # This test requires gnome-keyring which requires running dbus
 make_check_args="--deselect=tests/imports/test_parse.py"
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools python3-requests python3-yaml python3-zxcvbn"
 depends="pass python3-defusedxml python3-magic python3-cryptography
-python3-SecretStorage python3-pykeepass python3-yaml"
-checkdepends="python3-pytest $depends"
```

Comment:
don't remove pytest

Reply:
oh, the pypi tarball doesn't have tests, in that case, could you just add
```
# pypi tarball doesn't contain tests and the github release doesn't contain pre-built manpages (is this true?)
# <link to sgn's issue>
make_check=no
```

Reply:
Yes the first iteration did build from github and it requires pandoc to build markdown man files

```
ls -l share/man/man1/
-rw-r--r-- 1 alan alan 20046 Sep 10 23:19 pass-import.md
-rw-r--r-- 1 alan alan  8795 Sep 10 23:19 pimport.md